Makefile

```diff
@@ -317,6 +317,63 @@ clean-volumes:
 compose:
 	$(foreach MODULE, $(CANDIG_MODULES), $(MAKE) compose-$(MODULE);)
 
+#-- Temp --
```

Comment:
I assume this section will be removed before the PR is ready for merge...

Reply:
yes